server/server.go

```diff
@@ -249,17 +256,20 @@ func New(cfg Config) (*Server, error) {
 	if cfg.RegisterInstrumentation {
 		RegisterInstrumentation(router)
 	}
-	httpMiddleware := []middleware.Interface{
-		middleware.Tracer{
-			RouteMatcher: router,
-		},
-		middleware.Log{
-			Log: log,
-		},
-		middleware.Instrument{
-			Duration:     requestDuration,
-			RouteMatcher: router,
-		},
+	httpMiddleware := []middleware.Interface{}
+	if !cfg.DisableGeneratedHTTPMiddleware {
```

Comment:
I think I would put this `if` on the `append` not the static declaration.

Reply:
Makin standard middleware optional was moved out of this PR. So this code is no longer there.